### ISSUE TITLE
Feature request: Include more information when error message appears

### ISSUE THREAD

**Issue by erinyoung** [sanger-pathogens/Roary#460]
I have a feature request!

TL;DR: Please include sample and gene with issues when the 
```
--------------------- WARNING ---------------------
MSG: Got a sequence without letters. Could not guess alphabet
---------------------------------------------------
```
is displayed.

Here's the long version of the story:

For _reasons_, I needed to align some Acinetobacter sequences to create a phylogenetic tree. I also wanted to include some sequences that I found through blast. The samples were SAMN02470137, SAMN02894434, SAMN04014897, SAMN04014904, SAMN04096368, SAMN04096370, SAMN06650245, SAMN07945345, SAMN07977762, SAMN09906510, SAMN10411605. I downloaded the fasta files associated with each sample (meaning, both the chromosomal fasta as well as associated plasmids into one multifasta for each sample) and ran them through Prokka. Now I want to align them via Roary, and it's a pain to try to figure out which sequence is causing the error. I feel like in an ideal world, Roary would be able to tell which sample was problematic and where. 

Here's the command I used:
```
roary -p 45 -f Roary_out -e -n -qc -k /home/kraken_mini_db/minikraken_20141208 *.gff
```

Here's the summary_statistics.txt text for this set of samples:
```
Core genes	(99% <= strains <= 100%)	1568
Soft core genes	(95% <= strains < 99%)	0
Shell genes	(15% <= strains < 95%)	4378
Cloud genes	(0% <= strains < 15%)	33220
```

and the qc_report.csv text:
```
Sample,Genus,Species
SAMN02470137,Acinetobacter,Acinetobacter baumannii
SAMN02894434,Acinetobacter,Acinetobacter baumannii
SAMN04014897,Acinetobacter,Acinetobacter baumannii
SAMN04014904,Acinetobacter,Acinetobacter baumannii
SAMN04096368,Acinetobacter,Acinetobacter baumannii
SAMN04096370,Acinetobacter,Acinetobacter baumannii
SAMN06650245,Acinetobacter,Acinetobacter baumannii
SAMN07945345,Acinetobacter,Acinetobacter baumannii
SAMN07977762,Acinetobacter,Acinetobacter baumannii
SAMN09906510,Acinetobacter,Acinetobacter baumannii
SAMN10411605,Acinetobacter,Acinetobacter baumannii
```

**Comment by Ikkik** [comment]
Not quite. #229 had an error with his sequences being too distantly related.

I'm just pointing out that the error message is not as beneficial to the end user as it could be.
